package.json

```diff
@@ -49,7 +49,7 @@
     "test-detect": "mocha test/detect",
     "test-browser": "mocha test/browser",
     "test-parser": "mocha test/parser",
-    "view-browser": "npm run build_and_test_browser && http-server ./build/"
```

Comment:
Why wouldn't it be rooted into build?

Reply:
I was a bit confused as to which build I had to use. I can make the change.

Reply:
sorry, I was a bit confused. I changed the script a bit so that it focuses on the online tools created by the highlight.js team. The whole purpose of this pr, is so that new contributors can easily use a one script run command to access the online dev tools.